doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Usage & billing"
+description: "Understand the billing model of Materialize, and learn best practices for cost control."
+menu:
+  main:
+    parent: "manage"
+    weight: 50
+---
+
+Materialize determines billing based on your compute and storage usage.
+Materialize bills per second based on the [cluster(s)](/concepts/clusters/) you
+provision for your workloads. Each cluster is a pool of resources (CPU, memory,
+and scratch disk space) that must stay up and running to continually provide you
+with always-fresh results.
+
+## Compute
+
+In Materialize, [clusters](/concepts/clusters/) are pools of compute resources
+(CPU, memory, and scratch disk space) for running your workloads, such as
+maintaining up-to-date results while also providing strong [consistency
+guarantees](/get-started/isolation-level/). The credit usage for a cluster is
+measured at a one second granularity.
+
+You must provision at least one cluster to power your workloads. You can then
+use the cluster to create the object ([indexes](/concepts/indexes/) and
+[materialized views](/concepts/views/#materialized-views)) that provide
+always-fresh results. In Materialize, both indexes and materialized views are
+incrementally maintained when Materialize ingests new data. That is, Materialize
+performs work on writes such that no work is performed when reading from these
+objects.
```

Comment:
```suggestion
You must provision at least one cluster to power your workloads. You can then
create objects that provide always-fresh results in that cluster ([indexes](/concepts/indexes/)
and [materialized views](/concepts/views/#materialized-views)). In Materialize,
both indexes and materialized views are incrementally maintained when
Materialize ingests new data. That is, Materialize performs work on writes such
that no work is performed when reading from these objects.
```

Reply:
Needs at least object->objects.

Reply:
QQ: For the materialized views ... that will provide always-fresh results across clusters, yes?

Reply:
Then, will go with object -> objects.